apps/www/pages/pricing/index.tsx

```diff
@@ -292,7 +300,15 @@ export default function IndexPage() {
                           )}
                         </div>
                         <a href={plan.href}>
```

Comment:
Probably should be like this.
otherwise it will get focussed twice when keyboard tabbing.

```js
<Button asChild>
  <a>
  //
```